lib/book.rb

```diff
@@ -1,2 +1,15 @@
+class Book 
+  attr_reader :title
+  attr_accessor :author
+  attr_accessor :page_count
+  attr_accessor :genre
```

Comment:
Remember, you can chain these:
```suggestion
  attr_accessor :author, :page_count, :genre
```